### ISSUE TITLE
Future : change method for multi-select plotting

### ISSUE THREAD

**Issue by kkoch** [glos/habs_portal#105]
It would be more intuitive for the user if the multi-chart selection were more of a check box next to the nav item.  It would make it easier for the user to see what they've selected, to make more choices and deselect items.   Something like the attached mockup image (but possibly with with checkboxes on the right)
![multiselect](https://cloud.githubusercontent.com/assets/6392982/7122425/49a4926e-e1e9-11e4-88fd-d69b68682b2a.JPG)

**Comment by lukecampbell** [comment]
LoE: >4 days (but in the works as part of OceansMap v2)

**Comment by everhamme** [comment]
When doing multi station plotting is there a way to add the station name to the label?

**Comment by lukecampbell** [comment]
> When doing multi station plotting is there a way to add the station name to the label?

Yes, and it's been done in a recent commit. You should see it on production shortly.
